src/logging/default-logger.ts

```diff
@@ -175,6 +175,10 @@ export class DefaultLogger implements Logger {
             // fairly noisy. Maybe in a verbose mode.
             break;
           }
+          case 'terminated': {
+            console.error(`💀${prefix} Terminated`);
```

Comment:
I encounter the term 'killed' more often than 'terminated' when talking about ending an ongoing process

Reply:
Done. New terms: `#killRunningScripts`, `Killed` event.